### PR TITLE
UI polishing

### PR DESCRIPTION
**Pull request by juliushaertl**
From https://github.com/nextcloud/server/issues/10094

- [x] Top image & logo missing
- [x] Next button needs more padding, to 44px min clickability
- [ ] ~Icons on some entries missing~
- [ ] First slide should have a nice picture in content instead of text. (Like on Nextcloud flyer)
- [x] Fix mobile rendering
- [x] Adjust icons to new server classes after the files got unified in https://github.com/nextcloud/server/pull/9984

### CODE REVIEW

**Comment by juliushaertl** [comment]
>￼ Icons on some entries missing

@jancborchardt I cannot see any missing icon there, can you point me to what exactly is missing

>￼ First slide should have a nice picture in content instead of text. (Like on Nextcloud flyer)

@jancborchardt Do you have that image at hand somewhere? Or maybe @jospoortvliet 

And should this image replace the header area with the background image or the content, where we have the core values? Because then we would need to move the core values to a separate page.

**Comment by jancborchardt** [comment]
@juliushaertl sent you the link to the background image via Nextcloud Talk. ;)

I can actually not reproduce the images not loading, maybe another problem specific to Franks setup. But I found 2 new issues:

- Icons on first slide are not correctly shown on mobile – might be solved by simply using the image
- Scrolling is very slow cause it scrolls the whole page (see the whole page behind the modal being scrolled up)

screenshot from 2018-07-17 21-07-09

**Comment by jancborchardt** [comment]
And the image should replace the icons / text boxes on the bottom. And if I remember correctly (correct me if I’m wrong @jospoortvliet) we decided that we do _not_ insert another slide for it, because it’s just too much blabla that people will skip anyhow.

**Comment by juliushaertl** [comment]
@jancborchardt The icons about the core values of Nextcloud were actually the main requirement for the firstrunwizard rework. See https://github.com/nextcloud/firstrunwizard/pull/57 so I wonder why those are supposed to be dropped again. Also just using the image instead of the text/icons as a first slide doesn't provide any value to the user: 

But I like the idea of using that image, maybe as the header image:
image

**Comment by MorrisJobke** [comment]
@juliushaertl Should we get this in like it is now and finish the polishing later?

File: templates/personal-settings.php [hunk 1]
```diff
@@ -59,12 +59,12 @@
...
 		<a target="_blank" class="button" href="<?php p(link_to_docs('user-sync-calendars')) ?>">
 			<img class="appsmall appsmall-calendar svg" alt=""
-				 src="<?php p(image_path('core', 'places/calendar-dark.svg')); ?>" />
+				 src="<?php p(image_path('core', 'places/calendar.svg')); ?>" />
 			<?php p($l->t('Connect your calendar'));?>
 		</a>
 		<a target="_blank" class="button" href="<?php p(link_to_docs('user-sync-contacts')) ?>">
 			<img class="appsmall appsmall-contacts svg" alt=""
-				 src="<?php p(image_path('core', 'places/contacts-dark.svg')); ?>" />
+				 src="<?php p(image_path('core', 'places/contacts.svg')); ?>" />
```
Comment (MorrisJobke) [inline templates/personal-settings.php]: @juliushaertl I fixed the icons in here as well 😉

**Pull request merged**